runware/server.py

```diff
@@ -191,15 +237,20 @@ async def _handle_messages(self):
 
     async def send(self, msg: Dict[str, Any]):
         self.logger.debug(f"Sending message: {msg}")
-        if self._ws and self._ws.state is State.OPEN and not self._is_shutting_down:
-            await self._ws.send(json.dumps(msg))
+
+        async with self._send_lock:
```

Comment:
just curious why is it needed? websocket is already synchronized via internal queue